generator-jhipster-gomicro/package.json

```diff
@@ -0,0 +1,56 @@
+{
+  "name": "generator-jhipster-gomicro",
+  "version": "0.0.0",
+  "private": true,
+  "description": "Description for Gomicro",
+  "keywords": [
+    "yeoman-generator",
+    "jhipster-blueprint",
+    "jhipster-7"
+  ],
+  "license": "UNLICENSED",
+  "type": "module",
+  "imports": {
+    "#test-utils": "./test/utils.mjs"
+  },
+  "files": [
+    "generators"
+  ],
+  "scripts": {
+    "ejslint": "ejslint generators/**/*.ejs && ejslint generators/**/*.ejs -d '&'",
+    "lint": "eslint .",
+    "lint-fix": "npm run ejslint && npm run lint -- --fix",
+    "mocha": "mocha generators --no-insight --forbid-only",
+    "prettier:check": "prettier --check \"{,src/**/}*.{md,json,yml,html,js,ts,tsx,css,scss,vue,java}\"",
+    "prettier:format": "prettier --write \"{,src/**/}*.{md,json,yml,html,js,ts,tsx,css,scss,vue,java}\"",
+    "pretest": "npm run prettier:check && npm run lint",
+    "test": "npm run mocha --",
+    "update-snapshot": "npm run mocha -- --no-parallel --updateSnapshot"
+  },
+  "dependencies": {
+    "chalk": "4.1.2",
+    "generator-jhipster": "file://home/saiharsha/Desktop/generator-jhipster",
```

Comment:
change this
"generator-jhipster": "file://home/saiharsha/Desktop/generator-jhipster",
to
"generator-jhipster": "file:/usr/src/app/generator-jhipster",
as when we are building the image for the deployment it will have this as reference